### ISSUE TITLE
DoubleSided transmission broken with `WEBGL_multisampled_render_to_texture` (Oculus Browser)

### ISSUE THREAD

**Issue by mrxz** [mrdoob/three.js#28131]
### Description

When transmissive materials are set to `DoubleSided` the transmissive objects are rendered in two passes (see: https://github.com/mrdoob/three.js/pull/25502). The back side of the transmissive objects are rendered into the `transmissiveRenderTarget` while using the intermediate result of the `transmissiveRenderTarget`. This works as the target is multi-sampled and uses two textures under the hood.

However, when the `EXT_multisampled_render_to_texture` is supported by the browser, it is used instead of the two textures approach. In this case rendering the back-sides fails as source and destination textures are the same.
```
[.WebGL-0x20170a200]GL ERROR :GL_INVALID_OPERATION : glDrawElements: Source and destination textures of the draw are the same.
```

### Reproduction steps

1. Setup a scene with a double sided transmissive material
2. Open in a browser that supports `WEBGL_multisampled_render_to_texture`
3. Notice the back-side not being rendered and error(s) logged in the console

### Code

```js
const geometry = new THREE.SphereGeometry( 20, 64, 32 );

const material = new THREE.MeshPhysicalMaterial( {
	metalness: 0,
	roughness: 0,
	ior: 1.5,
	envMap: /* some equirectEnvMap */ hdrEquirect,
	transmission: 1.0,
	side: THREE.DoubleSide,
} );

mesh = new THREE.Mesh( geometry, material );
scene.add( mesh );
```

### Live example

The bug is present in both [`webxr_vr_sandbox`](https://threejs.org/examples/#webxr_vr_sandbox) and [`webgl_materials_physical_transmission`](https://threejs.org/examples/#webgl_materials_physical_transmission).

### Screenshots

| Without  `WEBGL_multisampled_render_to_texture` | With `WEBGL_multisampled_render_to_texture` |
|---------------| --------|
| ![image](https://github.com/mrdoob/three.js/assets/8823461/1bc0d36c-282e-4a15-a9b8-bf31408a389c) | ![image](https://github.com/mrdoob/three.js/assets/8823461/d4f658ab-d8d9-4387-9508-406ec6620191) |

### Version

r163

### Device

Headset

### Browser

Chrome

### OS

Android

**Comment by Mugen87** [comment]
Yeah, we were hoping to circumvent the feedback loop by using always using a multisampled render target, see #26177. It seems there is no way around a separate solution for the `EXT_multisampled_render_to_texture` case.
